setup/services.py

```
@@ -344,9 +364,7 @@ def start_services(args):
         if available_services.get("frontend", False):
             frontend_config = config.get_service_config("frontend")
```

Flake8 (7.3.0)

[error] local variable 'frontend_config' is assigned to but never used

(F841)